src/scenarios/shared/runner.py

```diff
@@ -62,11 +70,39 @@ def run(self):
             startup.runtests(**self.traits._asdict(),
                              scenariotypename=const.SCENARIO_NAMES[const.STARTUP],
                              apptorun=publishedexe(self.traits.exename))
-        # if testtype == 'sdk' and self.traits.sdk:
-        #     print("sdk")
-        #     startup = StartupWrapper()
-        #     startup.runtests(**self.traits._asdict(),
-        #         scenariotypename='Build No Changes')
-        #     # fix some other traits
-        #     startup.runtests(**self.traits._asdict(),
-        #         scenariotypename='Rebuild')
\ No newline at end of file
+        elif self.testtype == const.SDK:
+            startup = StartupWrapper()
+            envlistbuild = 'DOTNET_MULTILEVEL_LOOKUP=0'
+            envlistcleanbuild= envlistbuild+';'+'MSBUILDDISABLENODEREUSE=1'
```

Comment:
nit: to make this easier in the future, if you make the environment variables a list you can just join them together:
```
>>> ';'.join(['a','b','c'])
'a;b;c'
```
If we add one it's just adding to the list, then.

Reply:
good point!